Src/AutoFixture/Seed.cs

```diff
@@ -1,84 +0,0 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Reflection;
-
-namespace Ploeh.AutoFixture
-{
-    /// <summary>
-    /// Encapsulates a seed for a given type.
-    /// </summary>
-    /// <typeparam name="T">The type for which the seed applies.</typeparam>
-    public class Seed<T> : ICustomAttributeProvider

```

Comment:
Apparently, in d9bf1053d04742d588fcf59c76035bb4c6241325 I meant to move Seed.cs to the Kernel directory, but instead of moving the file, I copied it.

This file is now removed in *master*.